frontend/src/app/components/transactions-list/transactions-list.component.ts

```diff
@@ -78,7 +78,7 @@ export class TransactionsListComponent implements OnInit, OnChanges {
     const scrollTop = document.documentElement.scrollTop;
     if (scrollHeight > 0){
       const percentageScrolled = scrollTop * 100 / scrollHeight;
-      if (percentageScrolled > 90){
+      if (percentageScrolled > 80){
```

Comment:
I've done some testing and it might be better if we let the data refresh take place if the scroll percentage is greater than 60% of the page. This way the scrolling is smoother, without having to wait for the loading placeholder.

`if (percentageScrolled > 60){`